lib/Steps/Integrations/NextJs.ts

```diff
@@ -437,4 +457,58 @@ export class NextJs extends BaseIntegration {
     arr.splice(start, deleteCount, ...inserts);
     return arr;
   }
+
+  private async _mergeNextConfig(
+    destinationPath: string,
+    templatePath: string,
+    destinationDir: string,
+    templateFile: string,
+    configDirectory: string,
+    mergeableFilePath: string,
+  ): Promise<void> {
+    // if no next.config.js exists, we'll create one
+    if (!fs.existsSync(destinationPath)) {
+      fs.copyFileSync(templatePath, destinationPath);
+      green('Created File `next.config.js`');
+      nl();
+    } else {
+      const originalFilePath = path.join(
+        destinationDir,
+        this._spliceInPlace(templateFile.split('.'), -1, 0, 'original').join(
+          '.',
+        ),
+      );
+      // makes copy of original next.config.js
+      fs.writeFileSync(originalFilePath, fs.readFileSync(destinationPath));
+      await this._addToGitignore(
+        originalFilePath,
+        'Unable to next.config.original.js add to gitignore',
+      );
+
+      const mergedTemplatePath = path.join(
+        configDirectory,
+        'next.config.template.js',
+      );
+      // attempts to merge with existing next.config.js, if true -> success
+      if (mergeConfigFile(destinationPath, mergedTemplatePath)) {
+        green(
+          `Updated \`next.config.js\` with Sentry. The original next.config.js was saved as \`next.config.original.js\`.\n` +
+            "Information about Sentry's changes can be found at https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/",
+        );
+        nl();
+      } else {
+        // if merge fails, we'll create a copy of the `next.config.js` template and ask them to merge
+        fs.copyFileSync(templatePath, mergeableFilePath);
+        await this._addToGitignore(
+          mergeableFilePath,
+          'Unable to next.config.js template add to gitignore',
+        );
+        red(
+          `Unable to merge  \`next.config.js\`, so created \`${mergeableFilePath}\`.\n` +
+            'Please merge those files.',
```

Comment:
We could maybe be a little bit more instructive here. How about "Please integrate next.config.wizardcopy.js into your next.config.js or next.config.ts file".